web/workflowApi.go

```diff
@@ -14,7 +16,7 @@ var (
 )
 
 type workflowAPI struct {
-	topic string
+	id uint32
```

Comment:
I don't think we need there here. Unless this is being used elsewhere.

Reply:
I'm just leaving this struct empty for now then.